### DISCUSSION TITLE
Load config from k8s configmap

### DISCUSSION THREAD

**Discussion by qs-wang** [edgexfoundry/community#321] [Ideas]
While running on edge cluster, i.e. k3s, not not want to use centralized registry service. However use local file seems not ideal either. Would be a useful feature to support loading config from k8s configmap

**Comment by jiekechoo** [comment]
@qs-wang Here is a few examples about `configmap` in the `Helm Chart`. https://github.com/edgexfoundry/edgex-helm/tree/main/templates/configmaps

**Comment by cloudxxx8** [comment]
In each container settings, you can eliminate `-cp` and `--registry` by overwriting the `CMD` to disable the config provider and service registry
https://github.com/edgexfoundry/edgex-go/blob/2801c4a9184f74e54c01e3d1eec2799807029e6c/cmd/core-data/Dockerfile#L57

**Comment by qs-wang** [comment]
@jiekechoo @cloudxxx8 Thanks for the suggestions.  Appreciate!

I am writing my own helm chart for just including minimal set of services, and use config file only.  This is for my project which is running Edgex on edge with K3S. 

The repo is at https://github.com/tuxiot/edgex-k3s.
